src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
use object destructing
```suggestion
export const Person = ({ person }) => {
  const { name, .... } = person;
  
  return (
    <section className="Person">
```